docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
     restart: unless-stopped
     volumes:
       - "db_data:/var/lib/postgresql/data"
+    ports:
+      - "5432:5432"
```

Comment:
Opening the DB port to the internet is not a good idea - the bot can access it over the docker network without it being forwarded to the host. If you want to explicitly expose it *only* on the docker network, use the `preview` keyword instead of `port`. If you want PGAdmin access, it's probably best to just add a PGadmin container to the compose file.